src/widgets/medications/medication-order-basket.component.tsx

```diff
@@ -40,6 +41,8 @@ export default function MedicationOrderBasket(
     orderEdit: Boolean;
     order?: OrderMedication;
   }>({ orderEdit: false, order: null });
+  const medsRef = React.useRef<HTMLDivElement>(null);
```

Comment:
What's the purpose of this? I only see that we defined it, but we never read from it.

Reply:
removed